src/pages/Explorer/Category/ExplorerCategory.svelte

```diff
@@ -38,6 +38,25 @@
     items = items.filter((item) => getExplorerItem(item) !== itemToExclude)
   })
 
+  setContext('updateExplorerItem', (itemToUpdate, title, description, isPublic) => {
+    const itemIndex = items.findIndex((item) => getExplorerItem(item) === itemToUpdate)
+    const key = getExplorerItemProperty(items[itemIndex])
+    const item = items[itemIndex][key]
```

Comment:
@peratik looks like `itemToUpdate` and `items[itemIndex][key]` reference the same object. Do they?